mmdet/core/bbox/match_costs/match_cost.py

```diff
@@ -244,12 +244,18 @@ class DiceCost:
         pred_act (bool, optional): Whether to apply sigmoid to mask_pred.
             Defaults to False.
         eps (float, optional): default 1e-12.
+        naive_dice (bool, optional): If false, use the dice \
+                loss defined in the V-Net paper, otherwise, use the \
+                naive dice loss in which the power of the number in the
+                denominator is the first power instead of the second
```

Comment:
in which the denominator is calculated by the first power instead of the second power of mask values.